hello-electron/test/spec.js

```diff
@@ -0,0 +1,39 @@
+const Application = require('spectron').Application
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+const electronPath = require('electron')
+const path = require('path')
+
+chai.should()
+chai.use(chaiAsPromised)
```

Comment:
@muardle do you know what this does?

Reply:
@lgvalle The first line declares the preferred assertion style for Chai (in this case, [should](http://www.chaijs.com/api/bdd/)). The second line is required to integrate [Chai as Promised](https://github.com/domenic/chai-as-promised) with Chai.